apps/src/templates/teacherHomepage/TeacherHomepage.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const TeacherHomepage = React.createClass({
+  propTypes: {
+    children: React.PropTypes.oneOfType([
+      React.PropTypes.node,
+      React.PropTypes.arrayOf(React.PropTypes.node)
+    ])
+  },
+
+  render() {
+    const content = this.props.children;
+
+    return (
+      <div>
+        {content}
+      </div>
+    );
+  }
+});
```

Comment:
This component isn't really doing anything yet. Should the caller just use a div until more is needed?